docs/architecture/README.md

```diff
@@ -27,4 +27,7 @@ To suggest an ADR, please make use of the [ADR template](./adr-template.md) prov
 
 | ADR \#                                                    | Description                                           | Status   |
 |-----------------------------------------------------------|-------------------------------------------------------|----------|
-
+| [001](./adr-001-handler-implementation.md)                | IBC handlers implementation                           | Accepted |
```

Comment:
I didn't add a changelog for this since it doesn't relate to releases; i.e. IMO "updated ADRs" doesn't relate to a release, it's more a meta repo event. WDYT?

Reply:
Good point! :+1: